packages/config/src/config/vector.style.ts

```diff
@@ -1,5 +1,15 @@
+import { GoogleTms, Nztm2000QuadTms } from '@basemaps/geo';
+
 import { ConfigBase } from './base.js';
 
+/**
+ * Default Terrain exaggeration settings for different projection
```

Comment:
could we expand this to why they need the different numbers.

NZTM2000Quad is offset from WebMercatorQuad by 2 zoom levels so it needs to be approx 4x as large.

Reply:
Fixed

Reply:
This doesnt look like it has changed?

Reply:
My previous push failed, just pushed the changes.